File: C++/1716_totalMoney.cpp
class Solution {
public:
    int totalMoney(int n) {
        int total = 0;
        int dayPay = 1;
        for(int i = 1; i <= n; i++){
            total += dayPay;
            if(i%7 == 0)
                dayPay = (i/7);
            dayPay+=1;
        }
        return total;
        
    }
};

File: Python/1716_totalMoney.py
class Solution:
    def totalMoney(self, n: int) -> int:
        total = 0
        dayPay = 1
        for i in range(1,n+1):
            total += dayPay
            if(i%7 == 0):
                dayPay = (i//7)
            dayPay+=1
        return total
